### PR TITLE
[TRAFODION-1488] Master exec log files missing timestamps

### PR DESCRIPTION
**Pull request by robertamarton**

### CODE REVIEW

File: core/sql/qmscommon/QRLogger.cpp [hunk 1]
```diff
@@ -609,7 +623,13 @@ void QRLogger::log(std::string &cat,
...
         if ( myLevel == log4cxx::Level::getOff() )
           return;
-        if ( myLevel->toInt() < paramLevel->toInt() )
+
+        int_32 configuredLevel = myLevel->toInt();
+        int_32 requestedLevel = paramLevel->toInt();
+  
+        // If the configured logging level is greater (more restrictive) than
+        // the requested level, don't log. 
+        if ( configuredLevel > requestedLevel)
```
Comment (DaveBirdsall) [inline core/sql/qmscommon/QRLogger.cpp]: Thanks for these changes. The code is easier to understand now.

**Comment by DaveBirdsall** [comment]
Looks good to me.

**Comment by sureshsubbiah** [comment]
Looks good to me too. Thank you. Shall we merge this change or would you like to wait for more pairs of eyes to look at this Roberta?

**Comment by arvind-narain** [comment]
Thanks for the change Roberta. Looks good - do please go ahead and merge.

Just a note - CommonLogger does use the following (isGreaterOrEqual)- maybe this could be used in future changes rather than explicitly comparing levels - that way it will be independent of the actual enum values (difference between log4cxx and the previous logging mechanism used). 

result = ( myLevel != log4cxx::Level::getOff() && myLevel->isGreaterOrEqual(log4cxx::Level::getDebug()) )

**Pull request merged**